src/calendar-timeline/lib/classes/buckets/buckets.ts

```diff
@@ -1,3 +1,4 @@
+import { Granularity } from '../../enumertions/granularity.enum';
```

Comment:
I think the repository in which Granularity is should be named enumerations and not enumertions

Reply:
fixed